kuma/dashboards/views.py

```diff
@@ -71,8 +74,23 @@ def revisions(request):
             start_date = end_date - datetime.timedelta(seconds=seconds)
             query_kwargs['created__range'] = [start_date, end_date]
 
-    if query_kwargs:
-        revisions = revisions.filter(**query_kwargs)
+        authors_filter = filter_form.cleaned_data['authors']
+        if (not filter_form.cleaned_data['user'] != '' and
+           authors_filter not in ['', str(RevisionDashboardForm.ALL_AUTHORS)]):
+
```

Comment:
This feels awkward, but I don't have strong suggestions to improve it. Filters are just awkward.